### DISCUSSION TITLE
Add a custom link field

### DISCUSSION THREAD

**Discussion by bithuc** [NamelessMC/Nameless#2199] [Ideas]
when adding a link custom field, the link visible in will be cleaner.
when there is no custom field, my link does not display as expected.
I use a language other than English and it creates links by itself and it doesn't display exactly what I want it to do
I think not only the forum but all the links should have this link options field.
I don't understand why this link custom field doesn't appear on namelessmc, it's really important for a website. I hope this field appears on namelessmc soon because of its necessity and importance. Thank you for reading.
........................................................................................................................................................
- present
![loi diendan-1](https://user-images.githubusercontent.com/62389022/110502258-9f3cb780-812d-11eb-879f-e8bed67903fe.png)
- request
![loi diendan-2](https://user-images.githubusercontent.com/62389022/110503083-6a7d3000-812e-11eb-817d-a4b7cd349c03.png)
- For example
![loi diendan-3](https://user-images.githubusercontent.com/62389022/110503457-c8aa1300-812e-11eb-8bfc-7342b36c1cac.png)

**Comment by Derkades** [comment]
I think this is the underlying issue: https://github.com/NamelessMC/Nameless/issues/1683

**Comment by bithuc** [comment]
it is just a supplement. What I want is to add a link field so I can create a link according to my wishes
![loi diendan-3](https://user-images.githubusercontent.com/62389022/110633323-34e25080-81db-11eb-8150-5a461114b0e4.png)

**Comment by samerton** [comment]
I'm not sure this will be happening I'm afraid, we need an ID (the number) to know which forum you're trying to view.

You can use a custom page redirect if you want a vanity URL to point to a specific forum.
